readme.md

```diff
@@ -7,7 +7,7 @@
 
 Glimpse is an experimental **[npm package](https://www.npmjs.com/package/@glimpse/glimpse)** that gives you in-depth insights about the **client and server sides** of your **Node.js** apps. More **efficient debugging** means **faster development**. Best of all, it’s free.
 
-![Project Glimpse: Node Edition Screenshot](http://node.getglimpse.com/img/front-page/hero1.7848ce21.svg)
+![Project Glimpse: Node Edition Screenshot](/hero.png)
```

Comment:
See here: https://stackoverflow.com/questions/13808020/include-an-svg-hosted-on-github-in-markdown/21521184#21521184